CHANGELOG.rst

```diff
@@ -2,7 +2,11 @@ Unreleased
 ==========
 
 Changes since 0.4:
+
+* Added the opt-in classic_division fixer.
 * Updated the ``dict_six`` fixer to support ``six.viewitems()`` and friends.
+* Documentation corrections.
```

Comment:
No, it refers to https://github.com/python-modernize/python-modernize/commit/d0c0de2260c6cc26c2477ac7891263513e34178f